kuzzleio/sdk-jvm
# Add Gradle task to build a thin jar for Android/Kotlin project

## What does this PR do ?

Deliver a thin jar for Android Kotlin projects.
Change Getting Started Kotlin
Add Bintray publication for the thin jar






### How should this be manually tested?

The thin jar is built by default with `./gradlew build`

To build the fatJar use `./gradlew fatJar`

Initialize an Android/Kotlin project.
Add the fat jar as dependencies.
You should have "duplicate dependencies" issue.
Try to resolve it by adding those line to your application `gradle.build` file:

```groovy
configurations {
     cleanedAnnotations
     compile.exclude group: 'org.jetbrains'
     compile.exclude group: 'org.jetbrains.kotlin'
 }
```

If it does not resolve the problem add the thin jar, and add those dependencies in your application:

```groovy
    implementation("io.ktor:ktor-client-websockets:1.5.2")
    implementation("io.ktor:ktor-client-okhttp:1.5.2")
    implementation("io.ktor:ktor-client-cio:1.5.2")
    implementation("io.ktor:ktor-client-json:1.5.2")
    implementation("io.ktor:ktor-client-gson:1.5.2")
    implementation("io.ktor:ktor-client-serialization:1.5.2")
    implementation("com.google.code.gson:gson:2.8.5")
```

Resolves https://github.com/kuzzleio/sdk-jvm/issues/34

### Other Change

Bump `ktor` version from 1.3.2 to 1.5.2

### Boyscout

Fix Server Controller documentation
